climada/hazard/tc_tracks.py

```diff
@@ -1249,12 +1258,70 @@ def from_netcdf(cls, folder_name):
                                "whole life time.")
                 basin = track.basin
                 del track.attrs['basin']
-                track['basin'] = ("time", np.full(track.time.size, basin))
+                track['basin'] = ("time", np.full(track.time.size, basin, dtype="<U2"))
             data.append(track)
         tr = cls()
         tr.data = data
         return tr
 
+    def write_hdf(self, file_name, complevel=4):
```

Comment:
```suggestion
    def write_hdf5(self, file_name, complevel=4):
```
I think this is so far the most common naming of these methods in the CLIMADA code-base.

Reply:
Well spotted, I somehow missed this! :+1: